src/data/items.txt

```diff
@@ -9039,7 +9039,7 @@
 9011	fishin' hat	717158743	fishinhat.gif	hat	t,d	50
 9012	Trout Fishing in Loathing	339113101	book3.gif	usable	t	0	copies of Trout Fishing in Loathing
 9013	sonar fishfinder	463912431	evilometer.gif	offhand	t	0
-9014
+9014	luxury fly-tying workbench	689025155	flybench.gif	usable	t	0
```

Comment:
should have a `modifiers.txt` entry -- happy to do this if you like

alternatively if you run `test newitem 689025155` you can copy it out of the session file.